Converter/SemverConverter.php

```diff
@@ -40,7 +40,7 @@ public function convertVersion($version)
     public function convertRange($range)
     {
         $range = $this->cleanRange(strtolower($range));
-        $range = preg_replace('/(?<!\S)[vV]?\d+(?:\.\d+)?(?!\S)/', '${0}.*', $range);
+        $range = preg_replace('/(?<!\S)[vV]?\d+(?:\.\d+)?(?!\S)/', '${0}', $range);
```

Comment:
This means you will not actually be replacing anything, thus this whole line becomes pointless.